lib/vyasa_web/components/contexts/discuss.ex

```diff
@@ -56,6 +58,8 @@ defmodule VyasaWeb.Context.Discuss do
     |> assign(content_action: :index)
     |> init_sheaf_lattice()
     |> init_sheaf_ui_lattice()
+    |> init_drafting_context()
+    |> init_reply_to_context()
```

Comment:
main meat of this PR, same pattern as the read mode. 

any kind of latice state additions can be destructively adding (overwriting) because we haven't begun incrementally updating the lattice states this early in the lifecycle of the mode (@ init)